usaspending_api/search/v2/views/spending_by_award.py

```diff
@@ -516,6 +587,55 @@ def construct_es_response_for_prime_awards(self, response) -> dict:
             ),
         }
 
+    def construct_es_response_for_subawards(self, response) -> dict:
```

Comment:
Can we add/expand the type hints?
```suggestion
    def construct_es_response_for_subawards(self, response: Response) -> dict[str, Any]:
```